### DISCUSSION TITLE
Display the search results with the exact Title match on top

### DISCUSSION THREAD

**Discussion by allafridman** [microsoft-search/pnp-modern-search#3628] [General]
How can I push the search result with the exact match to the top of the search results list. If I search for "Spring Sale" ?
I get results with the values in the Title in the following order  "New Spring Sale", "Latest Spring Sale", and only then "Spring Sale"

**Comment by kasperbolarsen** [comment]
It sounds like XRANK could be an option,  see https://github.com/microsoft-search/pnp-modern-search/issues/1196

**Reply by allafridman** [reply]
Does not really work. I have a list with 4 items:
Spring Sale
New Spring Sale
Upcoming Spring Sale
Spring Sale Near You

Searching  for 'spring sale' 
Using this: (ListID:"2b66caae-eb50-4260-b47a-1b071deaa2e9" AND "{searchTerms}") XRANK(stdb=5) Title="{searchTerms}"
got 4 results:

Spring Sale Near You
Rank: 1001.8712716

Spring Sale
Rank: 1001.6682582

Upcoming Spring Sale
Rank:1001.6606143

New Spring Sale
Rank:1001.2143025

**Reply by wobba** [reply]
Try to increase the stdb value. Also, some random testing shows that `Title=` seems to run like `Title:`
 in KQL.

FQL seems to work, and you can add this to the refinement filter template:

`xrank("{searchTerms}",Title:equals("{searchTerms}"), stdb=5)`
